### DISCUSSION TITLE
[Will be Resolve in 2.6] 如何关闭表情包（GIF）功能

### DISCUSSION THREAD

**Discussion by AnzhiZhang** [walinejs/waline#1112] [Q&A] [answered]
我使用 hexo 和 Fluid 主题，我希望关闭此功能，注意到有博客实现了关闭：[KOTTORY.ME](https://kottory.me/posts/%E8%AE%B0%E4%B8%80%E6%AC%A1XBox%E6%89%8B%E6%9F%84%E9%80%81%E4%BF%AE/)
![image](https://user-images.githubusercontent.com/37402126/172862233-cc9eaf60-8947-4f07-8c2a-ac16bb3558df.png)

<details>
<summary>我已搜索文档</summary>
<img src=https://user-images.githubusercontent.com/37402126/172861286-6e4b60e4-1987-43c1-bf21-82923a5ac8c9.png>
<img src=https://user-images.githubusercontent.com/37402126/172861322-5876a717-933e-4d9e-b562-c7b45ae58e76.png>
</details>
<details>
<summary>和 Issues</summary>
<img src=https://user-images.githubusercontent.com/37402126/172861535-64a4d8b0-f985-406a-888a-524aec0d5051.png>
<img src=https://user-images.githubusercontent.com/37402126/172861608-2414a813-3359-488f-b400-830a88c498e5.png>
<img src=https://user-images.githubusercontent.com/37402126/172861835-41558636-09d2-4c73-a893-7e8899b3de6d.png>
</details>

请问如何关闭，谢谢！

**Comment by Mister-Hope** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/33315834/172862574-bb95bd32-2f9e-45ed-a2ce-172d638a0f48.png)

**Reply by AnzhiZhang** [reply]
注意到有博客实现了关闭：[KOTTORY.ME](https://kottory.me/posts/%E8%AE%B0%E4%B8%80%E6%AC%A1XBox%E6%89%8B%E6%9F%84%E9%80%81%E4%BF%AE/)，这个是自己魔改实现的吗

**Reply by Mister-Hope** [reply]
你用css隐藏的方式可以“禁用”任何功能，只需要把你不想看到的元素干掉就好了。

**Reply by AnzhiZhang** [reply]
谢谢，期待新版发布
